src/jco.js

```diff
@@ -34,6 +34,7 @@ program.command('componentize')
   .requiredOption('-w, --wit <path>', 'WIT path to build with')
   .option('-n, --world-name <name>', 'WIT world to build')
   .option('--aot', 'Enable Weval AOT compilation of JS')
+  .option('--weval-bin <path>', 'Specify a custom weval binary to use')
```

Comment:
I would have expected we'd need to set this somewhere in this PR to use it?

Reply:
Yes -- this PR actually isn't *quite* ready yet -- I need to make sure that installing `weval` with https://github.com/jaxxstorm/action-install-gh-release actually works. Will move this to draft while I work on the CI bits. 

The place this has to go is in the CLI test where we call `componentize` -- just haven't added it yet!

Reply:
Added and in-use now w/ the `--aot` option!